docs/reference/commandline/daemon.md

```diff
@@ -54,6 +54,7 @@ weight = -1
       --mtu=0                                Set the containers network MTU
       --disable-legacy-registry              Do not contact legacy registries
       -p, --pidfile="/var/run/docker.pid"    Path to use for daemon PID file
+      --raw-logs                             Output logs with timestamps and no coloring
```

Comment:
this doesn't match what you have at https://github.com/docker/docker/pull/18621/files#diff-a348195a1b645d5b477946b1efae728bR104